packages/wrangler/src/deployment-bundle/module-collection.ts

```diff
@@ -64,6 +65,23 @@ export const noopModuleCollector: ModuleCollector = {
 	},
 };
 
+// Extracts a package name from a string that may be a file path
+// or a package name. Returns null if the string is not a valid
+// Handles `wrangler`, `wrangler/example`, `wrangler/example.wasm`,
+// `@cloudflare/wrangler`, `@cloudflare/wrangler/example`, etc.
+export function extractPackageName(packagePath: string) {
+	if (packagePath.startsWith(".")) return null;
+
+	const match = packagePath.match(/^(@[^/]+\/)?([^/]+)/);
```

Comment:
This would match `c:\foo\bar`.

Reply:
Do you think I should explicitly just disallow `\` in the regex for now? Until this plugin in general handles backslashes for windows paths?

Reply:
Awesome, thanks! Let me know if you need anything else on my end before merge!